relay-server/src/services/processor.rs

```diff
@@ -2894,13 +2895,13 @@ impl UpstreamRequest for SendMetricsRequest {
                 // Request did not arrive, we are responsible for outcomes.
                 Err(error) if !error.is_received() => {
                     for (scoping, quantities) in self.quantities {
-                        utils::reject_metrics::<Vec<Bucket>>(
+                        utils::reject_metrics::<&Bucket>(
                             &self.outcome_aggregator,
+                            &self.metric_stats,
```

Comment:
Right now we propagate the `metric_stats` but it's not needed since we pass no buckets. This is done because in a follow up ticket we will have to see how we want to propagate upstream the volume of rejected `BucketView` values. The reason for why this doesn't work right now can be seen in a comment inside `reject_metrics`.